app/resources/messages.js

```diff
@@ -22,6 +22,13 @@ class Messages {
 		\nUnless you have a good reason to believe otherwise, you should not proceed.
 		\nYou can click here if you'd like to proceed with the connection.`;
 	}
+
+	failEnterpriseOrgError(length) {
```

Comment:
`enterpriseOrgError` along sound good to me as the function name here.

Reply:
Alright, no problem. I'll change that.